### PR TITLE
Remove CI Stage metrics

### PR DESCRIPTION
**Pull request by anders-swanson**
These metrics are being replaced by another system.

### CODE REVIEW

**Review by bhatpmk — APPROVED** [review]

File: ci/oke-ocidns/Jenkinsfile [hunk 1]
```diff
@@ -243,8 +241,6 @@ pipeline {
                         // derive the prefix for the OKE cluster
                         OKE_CLUSTER_PREFIX = sh(returnStdout: true, script: "${WORKSPACE}/ci/scripts/derive_oke_cluster_name.sh").trim()
                     }
-                    // Derive Kubernetes version, which is used to set the value for a label in the metrics emitted by the tests
-                    env.K8S_VERSION_LABEL = sh(returnStdout: true, script: "${WORKSPACE}/ci/scripts/derive_kubernetes_version.sh ${params.OKE_CLUSTER_VERSION}").trim()
```
Comment (kminder) [inline ci/oke-ocidns/Jenkinsfile:247]: Does the new exporter mechanism include this information?

**Review by kminder — APPROVED** [review]
Approved.

File: ci/oke-ocidns/Jenkinsfile (hunk 1 above)
Comment (anders-swanson) [reply ci/oke-ocidns/Jenkinsfile:247]: No, it does not. Though I want to comment that I didn't find this data useful (never used it), and it was inconsistently used in the existing instrumentation.

**Review by kminder — DISMISSED** [review]
Approved.

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**